src/renderers/common/Renderer.js

```diff
@@ -1260,6 +1260,7 @@ class Renderer {
 		frameBufferTarget.scissorTest = this._scissorTest;
 		frameBufferTarget.multiview = outputRenderTarget !== null ? outputRenderTarget.multiview : false;
 		frameBufferTarget.resolveDepthBuffer = outputRenderTarget !== null ? outputRenderTarget.resolveDepthBuffer : true;
+		frameBufferTarget.autoAllocateDepthBuffer = outputRenderTarget !== null ? outputRenderTarget.autoAllocateDepthBuffer : false;
```

Comment:
I think we should rename `setOutputRenderTarget()` to `_setXROutputRenderTarget()`. `autoAllocateDepthBuffer` only exists for `XRRenderTarget` and `setOutputRenderTarget()` is only relevant for XR. We should not allow to use this method as a part of the public API.

Reply:
Agreed 👍